scripts/docker/docker-entrypoint.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+set -e
```

Comment:
Could you add a comment at the top explaining what this script does? I think is important to enphasise that it only replaces existing values but won't add new ones.

Reply:
updated. please review

Reply:
Didn't see the comment, where did you add it?

Reply:
added in the PR description under `changes`

Reply:
thank you